pipeline_dp/accumulator.py

```diff
@@ -352,11 +352,13 @@ class SumParams(AccumulatorClassParams):
 class SumAccumulator(Accumulator):
 
     def __init__(self, params: SumParams, values):
-        self._sum = sum(values)
         self._params = params
+        self._sum = np.clip(values, self._params.mean_var_params.low,
```

Comment:
A test fails because of access to `mean_var_params` at this moment, namely creating `mean_var_params` requires access `eps/delta` which are not available at that moment on Spark.

Could you please make [_aggregate_params](https://github.com/OpenMined/PipelineDP/blob/9eecae87e9ee34343ace737fa48fdc7c69dbd870/pipeline_dp/accumulator.py#L230) public (i.e. to remove _) and use `aggregate_params.low/high`?